requirements/pip.txt

```diff
@@ -31,9 +31,9 @@ pyyaml==5.4.1  # pyup: ignore
 Pygments==2.10.0
 
 # Basic tools
-redis==3.5.3
-kombu==5.2.1
-celery==5.2.0
+redis==4.0.2
+kombu==5.2.2
+celery==5.2.1
```

Comment:
we need to test these locally before merging this PR, last time there were errors that the CI didn't catch